google-cloud-clients/google-cloud-firestore/src/main/java/com/google/cloud/firestore/v1/FirestoreClient.java

```diff
@@ -97,13 +96,13 @@
  * methods:
  *
  * <ol>
- *   <li>A "flattened" method. With this type of method, the fields of the request type have been
+ *   <li> A "flattened" method. With this type of method, the fields of the request type have been
```

Comment:
I assume this is a bug in the generator and not in your CL. We should ideally remove the leading spaces.

Reply:
Correct.